roles/docker/tasks/main.yml

```diff
@@ -28,7 +28,20 @@
     state: started
 
 # Install docker-compose which also installs the docker python library required by some ansible docker modules
-- name: install docker-compose
+- name: install docker
   pip:
-    name: "docker-compose"
+    name: "docker"
```

Comment:
Quotes are not necessary here. Also, we have already had a docker role in the public role, can't we just import that one as a dependency?

Reply:
This is the docker role. The role wasn't working for me on both debian and centos but with these changes it is working on both

Reply:
Oh my bad, I didn't notice this is the role.